notebooks/community/generative_ai/bq_dataframes_llm_code_generation.ipynb

```diff
@@ -0,0 +1,891 @@
+{
```

Comment:
<p>should be 2023</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done